core/src/Text/Interpolation/Nyan/Core/Internal/Parser.hs

```diff
@@ -73,6 +77,12 @@ setIfNew desc new (OptionChanged ch, old)
   | old == Just new = fail $ "Switch option `" <> desc <> "` is set redundantly"
   | otherwise = return (OptionChanged True, Just new)
 
+setCommenting :: SwitchesOptionsSetter m => Bool -> m ()
+setCommenting enable = do
+  opts <- get
+  res <- setIfNew "allow comments" enable (commentingB opts)
```

Comment:
This naming is a bit off. How about `comments handling`?

Reply:
agreed, that should definitely be a noun